src/NorthwindWebsite/API/Controllers/CategoriesController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using NorthwindWebsite.Services.Interfaces;
+
+namespace NorthwindWebsite.API.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class CategoriesController : ControllerBase
+{
+    private readonly ICategoryService _categoryService;
+
+    public CategoriesController(ICategoryService categoryService)
+    {
+        _categoryService = categoryService;
+    }
+
+    [HttpGet(Name = nameof(GetCategories))]
```

Comment:
Is `Name` parameter required?

Reply:
Have you tried `URL.Action`? https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/routing?view=aspnetcore-6.0#generating-urls-by-action-name